prisma/migrations/20240319234708_reference_measurements/migration.sql

```diff
@@ -67,7 +67,7 @@ VALUES
     (180692, 'antler point count', 0, 10000, NULL, NULL),
     (202423, 'age', 0, 10000, NULL, 'The number of years that the animal has been alive for'),
     (202423, 'offspring count', 0, 10000, NULL, NULL),
-    (202423, 'tail length', 0, 10000, 'centimeter', NULL);
+    (202423, 'tail length', 0, 10000, 'centimeter', NULL); --This entry says that all animals have tails. This needs to change
```

Comment:
You can drop this change and move the comment / SQL into the measurement_values migration.